src/sentry/replays/endpoints/project_replay_details.py

```diff
@@ -25,16 +36,42 @@ class ReplayDetailsPermission(ProjectPermission):
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Replays"])
 class ProjectReplayDetailsEndpoint(ProjectEndpoint):
+    """
+    The same data as OrganizationReplayDetailsEndpoint, except project is required.
+    Query for a specific replay to fetch or to deleted.
+    """
+
     owner = ApiOwner.REPLAY
     publish_status = {
-        "DELETE": ApiPublishStatus.UNKNOWN,
-        "GET": ApiPublishStatus.UNKNOWN,
+        "DELETE": ApiPublishStatus.PUBLIC,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
     permission_classes = (ReplayDetailsPermission,)
 
+    @extend_schema(
+        operation_id="Retrieve a Replay Instance",
```

Comment:
could we rename this to something different since we already have the org endpoint titled as this?
<img width="231" alt="SCR-20240207-ouzo" src="https://github.com/getsentry/sentry/assets/56095982/614b1e9c-9db5-4303-a245-a8d409a43502">

Reply:
yes, it's also a naming collision!